README.md

```diff
@@ -106,4 +126,25 @@ This option allows control over the alignment of the image on the page.
 
 This option allows you to select a particular page from a draw.io file to
 create the image from. Note: an invalid page-index will revert to one of the
-other valid pages (draw.io binary functionality)
\ No newline at end of file
+other valid pages (draw.io binary functionality)
+
+### Scale
+- *Formal Name*: `:scale:`
+- *Default Value*: `drawio_default_scale` set in conf.py
+- *Possible Values*: any positive integer
+
+This simply scales the size of the output image. By default draw.io *usually* 
+outputs relatively low resolution images, this setting can be used to remedy that.
+This overrides the `drawio_default_scale` set in conf.py for this specific diagram.
+If either width or height are set for a given image, scale will have no effect on
+the generated image.
+
+### Transparency
+- *Formal Name*: `:transparency:`
+- *Default Value*: `drawio_default_transparency` set in conf.py
+- *Possible Values*: `"true"` or `"false"`
+
+This changes the background transparency for diagrams exported to `png` files.
+Will override `drawio_default_transparency` which was st in conf.py for this
+specific diagram. If this setting is specified while the output format is not
+`png` it will have no effect on the generated image
```

Comment:
please add empty last line and remove this change 
![image](https://user-images.githubusercontent.com/25718989/79514274-23ef9480-8089-11ea-9942-8cca77114ed4.png)

Reply:
No idea whats causing that change?

Reply:
GitHub shows trailing space - else, could be line endings.